cli/ballerina-cli-utils/src/main/ballerina/packaging_pull/packaging_pull.bal

```diff
@@ -1,11 +1,35 @@
-
 import ballerina/internal;
 import ballerina/io;
 import ballerina/mime;
 import ballerina/http;
 
 @final int MAX_INT_VALUE = 2147483647;
 @final string VERSION_REGEX = "(\\d+\\.)(\\d+\\.)(\\d+)";
+DefaultLogger logger;
+
+documentation {
+    This object denotes the default logger object used when pulling a package directly.
```

Comment:
if offset is public, need to add it as field in the documentation block

Reply:
Changed access modifier of offset to private with ea151c5